File: Front/ProEventos-App/src/app/models/PalestranteEvento.ts
import { Evento } from "./Evento";
import { Palestrante } from "./Palestrante";

export interface PalestranteEvento {
  id: number;
  palestranteId: number;
  palestrante: Palestrante;
  eventoId: number;
  evento: Evento;
}

File: Back/src/ProEventos.Domain/PalestranteEvento.cs
namespace ProEventos.Domain
{
    public class PalestranteEvento
    {
        public int Id { get; set; }
        public int PalestranteId { get; set; }
        public Palestrante Palestrante { get; set; }
        public int EventoId { get; set; }
        public Evento Evento { get; set; }
    }
}